lib/libutils/isoc/include/malloc.h

```diff
@@ -56,6 +59,13 @@ void mdbg_check(int bufdump);
 #define memalign(alignment, size) \
 		mdbg_memalign(__FILE__, __LINE__, (alignment), (size))
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wshadow"
+static inline void free(void *ptr) {
```

Comment:
Travis / checkpatch complains about this line, the `{` should be on a separate line.